retrofit2/src/main/java/com/linecorp/armeria/client/http/retrofit2/ArmeriaCallFactory.java

```diff
@@ -56,12 +56,12 @@
     private final Map<String, HttpClient> httpClients = new ConcurrentHashMap<>();
     private final HttpClient baseHttpClient;
     private final ClientFactory clientFactory;
-    private final BiFunction<String, ? super ClientOptionsBuilder, ClientOptionsBuilder> configurator;
+    private final BiConsumer<String, ClientOptionsBuilder> configurator;
```

Comment:
It's actually intentional to use `BiFunction` here. A user may want to just override every options and it's fine.